src/test/unit/jws.test.js

```diff
@@ -72,6 +73,16 @@ describe('JWS', () => {
             uri: 'https://someswitch.com:443/prefix/parties/MSISDN/12345678',
             body,
         };
+        testOptsData = {
```

Comment:
```suggestion
        // An axios-style request uses the `.url` and `.data` properties instead of the `.uri` and `.body` properties.
        testOptsData = {
```

Reply:
And perhaps a similar comment above `testOpts`, e.g.

> A "request"-style request uses the `.uri` and `.body` properties instead of the `.url` and `.data` properties